pi-manage

```diff
@@ -579,6 +580,17 @@ def createdb(stamp=False):
         migration_dir = os.path.dirname(os.path.abspath(p[0]))
         fm_stamp(directory=migration_dir)
     db.session.commit()
+    warnings.warn("The function createdb() will be removed in Version 4.0", DeprecationWarning)
+
+
+@manager.option('--stamp', '-s', help='Stamp database to current head revision.',
+                default=False, action='store_true')
+def create_tables(stamp=False):
+    """
+    Initially create the tables in the database. The database must exist
+    (an SQLite database will be created).
+    """
+    createdb(stamp=False)
```

Comment:
When calling the productive function, you need to pass only "stamp".

Otherwise it will always be "False".

Reply:
Done: a5b50932686161a11509b56b4a98ac3665d0d493